### PR TITLE
accesslog: work with Telemetry API

### PR DESCRIPTION
**Pull request by zirain**

### CODE REVIEW

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+title: Work with Telemetry API
+description: This task shows you how to configure Envoy proxies to send access logs with Telemetry API.
+weight: 10
+keywords: [telemetry,logs]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
+Telemetry API was introduced into Istio as a first-class API for a long time.
+Previously; you'd be configuring telemetry in the `meshConfig` section of Istio configuration.
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:11]:
```suggestion
Telemetry API has been in Istio as a first-class API for quite sometime now.
Previously users had to configure telemetry in the `MeshConfig` section of Istio configuration.
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Get start with Telemetry API
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:13]:
```suggestion
## Get started with Telemetry API
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+1. Move to Telemetry API with following configuration:
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:15]:
```suggestion
1. Enable access logging
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    - name: envoy
+    match:
+      mode: SERVER
+    disabled: true
+EOF
+{{< /text >}}
+
+1. Filter access log with CEL expression
+
+The following configuration will only display access log when response code is greater or equal to 500:
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:81]:
```suggestion
The following configuration displays access log only when response code is greater or equal to 500:
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  accessLogging:
+  - providers:
+    - name: envoy
+    disabled: true
+EOF
+{{< /text >}}
+
+1. Filter access log with workload mode
+
+You can disable inbound access log for `details` service with following configuration:
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:57]:
```suggestion
You can disable inbound access log for `details` service with the following configuration:
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+  - providers:
+    - name: envoy
+EOF
+{{< /text >}}
+
+The above example uses the built-in `envoy` access log provider, and we do not configure anything other than default settings.
+
+1. Disable access log for specific workload
+
+You can disable access log for `details` service with following configuration:
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:35]:
```suggestion
You can disable access log for `details` service with the following configuration:
```

File: content/en/docs/tasks/observability/logs/telemetry-api/index.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    filter:
+      expression: response.code >= 500
+EOF
+{{< /text >}}
+
+For more information, see [Use expressions for values](/docs/tasks/observability/metrics/customize-metrics/#use-expressions-for-values)
+
+## Work with OpenTelemetry provider
+
+Istio support send access log with [OpenTelemetry](https://opentelemetry.io/) protocol, see more [here](/docs/tasks/observability/logs/otel-provider).
```
Comment (kfaseela) [inline content/en/docs/tasks/observability/logs/telemetry-api/index.md:106]:
```suggestion
Istio supports sending access logs with [OpenTelemetry](https://opentelemetry.io/) protocol,  as explained [here](/docs/tasks/observability/logs/otel-provider).
```

**Comment by zirain** [comment]
cc @lei-tang @kyessenov

**Review by lei-tang — APPROVED** [review]

**Review by lei-tang — COMMENTED** [review]
LGTM. Thanks for adding this guide!

**Review by kfaseela — APPROVED** [review]

**Pull request merged**